bsdapi/BsdApi.py

```diff
@@ -148,9 +148,16 @@ def cons_getBulkConstituentData(self, format, fields, cons_ids=None, filter=None
         return self._makePOSTRequest(url_secure, query)
 
     def cons_setConstituentData(self, xml_data):
```

Comment:
@lowellkitchen I'm going to leave it for now, in case any client wants to do it both ways with the same library version (e.g. feature-toggling old vs. new). I've added the deprecation notice, and we can remove it completely when we deprecate the API path in December.